src/Microsoft.ML.Api/TypedCursor.cs

```diff
@@ -563,9 +563,9 @@ public static class CursoringUtils
             SchemaDefinition schemaDefinition = null)
             where TRow : class, new()
         {
-            // REVIEW: Take an env as a parameter.
-            var env = new ConsoleEnvironment();
-            return data.AsCursorable<TRow>(env, ignoreMissingColumns, schemaDefinition);
+            // REVIEW: Take this as a parameter, or else make it a method on he context itself.
+            var ml = new MLContext(42);
```

Comment:
>42 [](start = 35, length = 2)

is the number intentional?

Reply:
No, some copy-pasta, but surely just making the cursors seed deterministic isn't going to hurt anything I hope. 😄